src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.jsx

```diff
@@ -126,6 +126,12 @@ const GroupDetails = createReactClass({
           Sentry.withScope(scope => {
             Sentry.captureException(new Error('Project not found'));
           });
+        } else {
+          const locationWithProject = {...browserHistory.getCurrentLocation()};
+          if (!locationWithProject.query.project) {
```

Comment:
We should check that it's strictly not `undefined` (i.e. in future we may decide that empty string represents "all projects"0.